objgraph.py

```diff
@@ -287,6 +289,40 @@ def show_growth(limit=10, peak_stats={}, shortnames=True, file=None,
 
     The caveats documented in :func:`typestats` apply.
 
+    Example:
+
+        >>> growth(2)
+        [(tuple, 12282, 10), (dict, 1922, 7)]
+
+    .. versionadded:: 3.2.1
```

Comment:
Let's make this 3.3.0 -- [SemVer](https://semver.org) says new features require minor version increases.